flytekit/common/tasks/task.py

```diff
@@ -168,6 +170,36 @@ def register(self, project, domain, name, version):
             self._id = old_id
             raise
 
+    @_exception_scopes.system_entry_point
+    def fast_register(self, project, domain, name, digest, additional_distribution) -> str:
```

Comment:
should this be in the task or in SDKRunnableTask?

Reply:
@wild-endeavor mind weighing in?

Reply:
done

Reply:
@kumare3 when i moved this I realized it breaks SDK Presto tasks (https://github.com/lyft/flytekit/blob/fd57a48a87ee41b003f2901a04fa1da46d596046/flytekit/common/tasks/presto_task.py#L19) - what are your thoughts on reverting back? or should i add a fast_register to the presto tasks too?